x-pack/plugins/cloud_security_posture/server/plugin.ts

```diff
@@ -72,7 +73,10 @@ export class CspPlugin
     });
 
     initializeCspRules(core.savedObjects.createInternalRepository());
-    initializeCspTransformsIndices(core.elasticsearch.client.asInternalUser, this.logger);
+    initializeCspTransformsIndices(core.elasticsearch.client.asInternalUser, this.logger).then(
```

Comment:
we should install the transforms on package installation instead of plugin startup.
that way we can also have the transforms namespace aware

Reply:
opened a tech debt for that